web-components/src/components/combobox/ComboBox.ts

```diff
@@ -73,6 +73,8 @@ export namespace ComboBox {
     @property({ type: Boolean, attribute: "show-selected-count", reflect: true }) showSelectedCount = false;
 
     @property({ type: Number, attribute: false })
+    @internalProperty() 
+    private isWithLabel = false;
```

Comment:
can we rename isWithLabel to isOptGroup?

Reply:
DONE renamed